book/src/proptest/tutorial/compound-strategies.md

```diff
@@ -18,6 +19,11 @@ fn test_add() {
     let mut runner = TestRunner::default();
     runner.run(/* uhhm... */).unwrap();
 }
+# #[cfg(not(test))]
+# fn test_add() {
+#     let mut runner = TestRunner::default();
+#     runner.run(/* uhhm... */).unwrap();
+# }
```

Comment:
what's does adding a not test version of the same code do?

Reply:
so does this compile and run in the background make it appear to the user that the `#[test]` version ran where actually this did?

Reply:
Yeah, the actual doctest is the cfg not test fn. The test function is just for display purposes. There's probably a better way to DRY this up. I'll take another pass at it.